services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainConfiguration.java

```diff
@@ -24,11 +25,13 @@
 import tech.pegasys.teku.infrastructure.exceptions.InvalidConfigurationException;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.networks.Eth2Network;
 
 public class PowchainConfiguration {
   public static final int DEFAULT_ETH1_LOGS_MAX_BLOCK_RANGE = 10_000;
   public static final boolean DEFAULT_USE_MISSING_DEPOSIT_EVENT_LOGGING = false;
   public static final boolean DEFAULT_DEPOSIT_SNAPSHOT_STORAGE_ENABLED = false;
+  public static final boolean DEFAULT_DEPOSIT_SNAPSHOT_BUNDLE_ENABLED = false;
```

Comment:
We can probably just combine these two flags into a single `--Xdeposit-snapshot-enabled` flag just to avoid having too many different toggles that interact in strange ways.